paparazzi/src/main/java/app/cash/paparazzi/accessibility/AccessibilityRenderExtension.kt

```diff
@@ -83,8 +90,8 @@ public class AccessibilityRenderExtension : RenderExtension {
     if (this is AbstractComposeView) {
       // ComposeView creates a child view `AndroidComposeView` for view root for test.
       val viewRoot = getChildAt(0) as? ViewRootForTest
```

Comment:
1) can we remove the safe cast here?
2) also, I forgot/didn't realize that this was using `ViewRootForTest`.  This will likely block SDK work; can we obtain the SemanticNode accessibility information through another means, in a follow-up?